pydantic/generics.py

```diff
@@ -60,11 +60,14 @@ def __class_getitem__(  # type: ignore
             _generic_types_cache[(cls, params[0])] = created_model
         return created_model
 
-
-def concrete_name(cls: Type[Any], params: Tuple[Type[Any], ...]) -> str:
```

Comment:
@samuelcolvin Note moving the function from the module namespace to being a method is a breaking change -- I guess this should go into 1.0 if we move this.

If you don't want to introduce a breaking change at this point, I could add the function back in and just call the module-level function from the classmethod.

Reply:
I think this is fine, it's a pretty small breaking change, I can accept it into the second beta.